RawSpeed/DngDecoderSlices.cpp

```diff
@@ -143,6 +143,159 @@ my_error_throw (j_common_ptr cinfo)
   ThrowRDE("JPEG decoder error!");
 } 
 
+static void decodeFPDeltaRow(unsigned char * src, unsigned char * dst, size_t tileWidth, size_t realTileWidth, unsigned int bytesps, int factor) {
```

Comment:
TBN, it is not from `dcraw.c`
https://github.com/Beep6581/RawTherapee/blob/880d0305486d8d9f6a4bb71c9f42a45921bfe48e/rtengine/dcraw.patch#L3173